packages/core-compat-api/src/collectLegacyRoutes.tsx

```diff
@@ -167,6 +168,9 @@ export function collectLegacyRoutes(
     return () => String(currentIndex++);
   })();
 
+  // Placeholder plugin for any routes that don't belong to a plugin
+  const orphanRoutesPlugin = createPlugin({ id: 'orphan-routes' });
```

Comment:
i wonder if this type of thing might benefit from like a `core-` id prefix or so 🤷

Reply:
yep I actually had `converted-orphan-routes` for a bit, I'll switch back to that